torcheeg/models/cnn/fbcnet.py

```diff
@@ -207,4 +178,144 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
     def feature_dim(self):
-        return self.num_S * self.in_channels * self.stride_factor
\ No newline at end of file
+        return self.num_S * self.in_channels * self.stride_factor
+
+
+class FBMSNet(nn.Module):
```

Comment:
It appears that the network architecture of FBMSNet doesn't offer significant advantages. I suspect that its loss function plays a significant role in its performance.

Could you please create a trainer under https://github.com/torcheeg/torcheeg/tree/main/torcheeg/trainers/center_loss_trainer.py?

You can refer to the core design of CenterLoss here: https://github.com/Want2Vanish/FBMSNet/blob/main/codes/centralRepo/CenterLoss.py